### PR TITLE
fixed #3599. add adjustTextFieldPosition function call to the visit method

### PR DESCRIPTION
**Pull request by zilongshanren**
add adjustTextFieldPosition function call to the visit method of CCEditBoxImplIOS to solve this issue

### CODE REVIEW

File: extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm [hunk 1]
```diff
@@ -614,7 +614,9 @@ static CGPoint convertDesignCoordToScreenCoord(const Point& designCoord, bool bI
 
 void EditBoxImplIOS::visit(void)
 {
-    
+    if (nullptr != _systemControl) {
+        this->adjustTextFieldPosition();
```
Comment (dumganhar) [inline extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm]: After thinking a while, I think we need to use a schedule to update the position of `UITextField`.
And the interval should be 0.5s probably.

Comment (zilongshanren) [inline extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm]: Ok, I will use the scheduler instead.

Comment (dumganhar) [inline extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm]: Thanks.

**Comment by dumganhar** [comment]
Thanks

**Comment by zilongshanren** [comment]
It's my honor:)

**Pull request merged**